src/current/_includes/releases/v23.1/v23.1.0-alpha.1.md

```diff
@@ -112,10 +110,8 @@ Release Date: December 19, 2022
 - CockroachDB now prioritizes non-voters in voter additions, meaning that when selecting a store to add a voter on (in the allocator), CockroachDB will prioritize candidate stores that contain a non-voter replica higher. This reduces the number of snapshots that need to be sent over the WAN. [#89650][#89650]
 - CockroachDB now uses response data rather than just the request span in the [load-based splitter](https://www.cockroachlabs.com/docs/v23.1/load-based-splitting) to pass more accurate data about the keys iterated over to the load splitter to find a suitable split key, enabling the load splitter to find a split key under heavy range query workloads. [#89217][#89217]
 - Added the `replicas.leaders_invalid_lease` metric, which indicates how many replicas are Raft group leaders but holding invalid leases. [#91179][#91179]
-- SQL tenants now support the [HTTP endpoint](https://www.cockroachlabs.com/docs/v23.1/cluster-api) under `/api/v2/sql` which allows the caller to execute an HTTP request containing SQL statements to execute. The JSON response contains the results. This endpoints works identically as on a non-tenant server, except that it naturally scopes to the target tenant for SQL execution. [#91323][#91323]
```

Comment:
@dikshant Please review only this removal. :)

Reply:
LGTM!